cruft/_commands/create.py

```diff
@@ -52,7 +51,7 @@ def create(
             generate_files(
                 repo_dir=cookiecutter_template_dir,
                 context=context,
-                overwrite_if_exists=overwrite_if_exists,
+                overwrite_if_exists=True,
```

Comment:
Feature request: Could we check for a git folder in the ouput dir, then warn if there isn't one there? Rolling back changes without git tracking could be a major pain, and I imagine a difficult support request to resolve when one of the AI engineers inevitably loses their project to it

Reply:
This is a great idea if we end up having to go down this route